components/Header.tsx

```diff
@@ -12,7 +12,7 @@ const Header = () => {
   ];
 
   return (
-    <header className="relative z-20 shadow-md w-full h-32 flex items-center justify-center">
+    <header className="z-50 shadow-md w-full h-32 flex items-center justify-center fixed md:relative top-0 left-0">
```

Comment:
Does this need to be `z-50`? If not, please change back to `z-20` in order to keep project-wide z-index usage tidy. I believe the only other z-index declared in this stacking context is `z-10`.